scripts/tundra/tools/msvc-vs2017.lua

```diff
@@ -0,0 +1,8 @@
+
+module(..., package.seeall)
+
+local vscommon = require "tundra.tools.msvc-vscommon-next"
+
+function apply(env, options)
+  vscommon.apply_msvc_visual_studio("2017", env, options)
```

Comment:
I've changed the versioning from a point release to a year, I'm assuming a Visual Studio 2019 or 2020 in the future and it will then use a similar directory layout, we use to use version numbers like 13.0 and 14.0 but those were used to search the registry which we no longer do, so this change reflects the directory layout.